src/rpc/util.h

```diff
@@ -106,17 +106,33 @@ struct RPCArg {
     std::string ToDescriptionString(bool implicitly_required = false) const;
 };
 
+struct RPCResult {
+    const std::string m_result;
+
+    RPCResult(
```

Comment:
Should be `explicit`? :-)

All reviewers of this PR: please review PR #14505 and make the `explicit` checking an automatic Travis check :-)

Reply:
IMO we could ignore explicit here, this is utility code, nothing to worry about..

Reply:
IMO we should follow the recommendation of our developer notes:

> - By default, declare single-argument constructors `explicit`.
>   - *Rationale*: This is a precaution to avoid unintended conversions that might
>   arise when single-argument constructors are used as implicit conversion
>   functions.

:-)